maths/miller_rabin.py

```diff
@@ -1,14 +1,14 @@
 import random
 
-from .binary_exp_mod import bin_exp_mod
+from binary_exp_mod import bin_exp_mod
```

Comment:
```suggestion
from .binary_exp_mod import bin_exp_mod
```
This is an incorrect way of importing in a namespace package. Please research first.